network-api/networkapi/buyersguide/templates/primary_nav.html

```diff
@@ -1,40 +1,25 @@
 {% extends "partials/primary_nav.html" %}
 
-{% load i18n %}
+{% load bg_nav_tags i18n %}
 
-{% block narrow_screen_menu %}
-  {% comment %}
-    This block can actually be empty becasue PNI doesn't show nav newsletter form on mobile.
-    However, our current implementation of nav-newsletters.js requires the following DOM nodes
-    to exist in order for desktop nav newsletter form to kick off properly.
-    Clean-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/4686
-  {% endcomment %}
-  <div class="narrow-screen-menu-background">
-    <div class="narrow-screen-menu-container d-md-none">
-      <button class="btn btn-secondary btn-newsletter">{% trans "Newsletter" %}</button>
-    </div>
+{% block nav_logo %}
+  <div class="d-flex align-items-center py-0 py-md-3 logo-section">
+    <a href="/" class="moz-logo mb-0 mr-3"><span class="sr-only">Mozilla</span></a>
+    <p class="mb-0 h3-heading flex-wrap">
+      <a class="link-back" href="{% url 'buyersguide-home' %}">{% trans "*privacy not included" context 'Buyer’s guide name. This can be localized. This is a reference to the “*batteries not included” mention on toys.' %}</a>
+    </p>
   </div>
 {% endblock %}
 
-{% block burger_and_logo %}
-  {# we do not want the burger and logo from the original nav here #}
-{% endblock %}
-
-{% block wide_screen_menu %}
-  <div class="d-flex align-items-center justify-content-between">
-    <div class="d-flex align-items-center py-0 py-md-3 logo-section">
-      <a href="/" class="moz-logo mb-0 mr-3"><span class="sr-only">Mozilla</span></a>
-      <p class="mb-0 h3-heading flex-wrap">
-        <a class="link-back" href="{% url 'buyersguide-home' %}">{% trans "*privacy not included" context 'Buyer’s guide name. This can be localized. This is a reference to the “*batteries not included” mention on toys.' %}</a>
-      </p>
-    </div>
-  </div>
+{% block narrow_screen_nav_links %}
+{% url 'buyersguide-home' as home_url %}
+<div><a class="{% bg_active_nav request.get_full_path home_url %}" href="{{ home_url }}">{% trans "Privacy Not Included" %}</a></div>
+{% include "./fragments/pni_nav_links.html" with pre="<div>" post="</div>" %}
```

Comment:
I wanted to name this new file `nav_link.html`. But if I did so, it would include `templates/fragments/nav_links.htm` (nav_links.html for the main Foundation site) instead of the new file I created. It's super weird and I can't figure out why.

Reply:
The only thing I can think of is that we load buyersguide last as list of installed Django apps, and templates are copied over into a "static" pool, but without appname unless that's in the file tree for the template folder(s), so it might just see the main `nav_links.html` and then when it sees the buyersguide one, go "I already know a file by this name"... =x

We solve this for wagtail by having that explicit dir, so you see:
```
networkapi
 |-wagtailpages
 |  |- ...
 |  |- templates
 |     |- wagtailpages (again)
 |        |- blocks
 |        |- fragments
 |        |- ...
```

but for buyersguide we skip that extra level.

Reply:
I've filed https://github.com/mozilla/foundation.mozilla.org/issues/5543

Reply:
Got it! Thanks for investigating it!